### PR TITLE
Update Self Registration to Add Trusted Origin

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Update self registration
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-367827](https://oktainc.atlassian.net/browse/OKTA-367827)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/configure-self-service-registration-policy/index.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ When you enable a self-service registration policy, Okta enforces uniqueness for
         * **User dashboard:** Select to redirect users to your org's Okta homepage.
         * **Custom URL** Select and then enter the URL where you want the authorization server to redirect your users, such as to your custom app or portal.
 
+    > **Note:** Make sure the domain for the custom URL that you've configured for the Default Redirect is added as a [Trusted Origin](/docs/guides/enable-cors/overview/) for redirects.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/configure-self-service-registration-policy/index.md:54]:
```suggestion
    > **Note:** Ensure the domain for the custom URL that you've configured for the Default redirect is added as a [Trusted Origin](/docs/guides/enable-cors/overview/) for redirects.
```

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/configure-self-service-registration-policy/index.md:54]: I think our style guide prefers "ensure". Also, using "redirect" to reflect the previous parameter name.

**Review by vanngo-okta — CHANGES_REQUESTED** [review]
small nit

File: packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/configure-self-service-registration-policy/index.md (hunk 1 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/configure-self-service-registration-policy/index.md:54]: Fixed. I also bolded "Default redirect" to reflect the parameter name.

**Review by vanngo-okta — APPROVED** [review]
LGTM! 🚢

**Pull request merged**